solutions/fully-configurable/outputs.tf

```
@@ -79,3 +79,28 @@ output "cos_kms_key_crn" {
   description = "The CRN of the key management service (Key Protect) key used to encrypt the Cloud Object Storage bucket that the solution creates."
   value       = local.cos_kms_key_crn
 }
+
+output "next_steps_text" {
+  value       = "You can now access the watsonx.ai project to manage data, collaborate, monitor resources and build prompts."
```

```suggestion
  value       = "The watsonx.ai project is ready to work with data and other assets to accomplish a particular goal."
```

does it sound correct? Feel free to update if required.